### DISCUSSION TITLE
Configuration mySQL with Flowise in docker.

### DISCUSSION THREAD

**Discussion by claclin** [FlowiseAI/Flowise#1714] [Q&A]
Hi folks. I am trying to deploy Flowise in docker with mysql database but I get always the same error:

Error: ENOENT: no such file or directory, mkdir
Code: ENOENT
I have modify the .env file as following:

#DATABASE_PATH=/root/.flowise
#APIKEY_PATH=/root/.flowise
#SECRETKEY_PATH=/root/.flowise
#LOG_PATH=/root/.flowise/logs

DATABASE_TYPE=mysql
DATABASE_PORT="3306"
DATABASE_HOST="localhost"
DATABASE_NAME="flowise"
DATABASE_USER="root"
DATABASE_PASSWORD="password"
DATABASE_SSL=true

Under the docker-compse I have adjusted the volume to my mariadb 

volumes:
- /home/user/mariadb/persistet:/root/.flowise

And I have adjusted the collation as appears on the video but I get alway the same error when I deploy it.

If I use the sqlite it works fine, but I have noticed that when I try to open the database with the HeidiSQL software I get the error of out of memory and I can´t open it. I can open only when I have the docket stopped...

Any help will be appreciate.

**Comment by chungyau97** [comment]
Hi @claclin, sqlite will work because it does not need to be installed inside docker.

I believe you need to refer this to setup your MySQL correctly.
https://docs.docker.com/get-started/07_multi_container/

**Reply by claclin** [reply]
Thanks I will check it.
